src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,18 @@
+.stars {
+  display: flex;
+
+  &__star {
+    background: $default-star-image no-repeat center;
+    width: $default-star-size;
+    height:  $default-star-size;
+    margin-right: 4px;
+  }
+
+  &--4 :nth-child(-n + 4) {
+    background-image: $active-star-image;
```

Comment:
What about other modifiers? Stars should work correctly with them too
![image](https://github.com/mate-academy/layout_product-cards/assets/96409563/744d1425-dcce-4a2c-8d99-b679c374772b)